index.html

```diff
@@ -48,31 +48,31 @@ <h2 class="support-text">Software Developer</h2>
             </div>
 
             <!-- Skills  -->
-            <section class="experience-container">
+            <div class="experience-container">
                 <ul class="language-container">
-                    <li><h5>Language</h5></li>
+                    <li class="language-experience">Language</li>
                     <li class="skill-language">JavaScript</li>
                     <li class="skill-language">Ruby</li>
                     <li class="skill-language">Html</li>
                     <li class="skill-language">Css</li>
                 </ul>                
                 <ul class="frame-works-container">
-                    <li><h5>Frameworks</h5></li>
+                    <li class="language-experience">Frameworks</li>
                     <li class="skill-frame">Bootstrap</li>
                     <li class="skill-frame">Ruby on Rails</li>
                     <li class="skill-frame">RsPes</li>
                     <li class="skill-frame">CapyBara</li>
                     <li class="skill-frame">Selenium</li>
                 </ul>
                 <ul class="skills-container">
-                    <li><h5>Skills</h5></li>
+                    <li class="language-experience">Skills</li>
                     <li class="skill-skills">Codekit</li>
                     <li class="skill-skills">Github</li>
                     <li class="skill-skills">Codepen</li>
                     <li class="skill-skills">Gitlab</li>
                     <li class="skill-skills">Terminal</li>
                 </ul>                
-            </section>
+            </div>
```

Comment:
- changing container from section to div. 
- fixing a heading level that is skipped. Users may be confused or experience difficulty navigating.